test/cljs/reitit/frontend/core_test.cljs

```diff
@@ -22,8 +32,10 @@
                 :path-params {}
                 :query-params {}
                 :path "/"
+                :fragment nil
                 :parameters {:query {}
-                             :path {}}})
+                             :path {}
+                             :fragment nil}})
```

Comment:
others default to `{}`, but fragment defaults to `nil`. `{}` would be a consistent default.

Reply:
It is a string, not a map. Could default to an empty string, though.